daemon/config/config.go

```diff
@@ -493,9 +493,7 @@ func findConfigurationConflicts(config map[string]interface{}, flags *pflag.Flag
 	if len(unknownKeys) > 0 {
 		unknownNamedConflicts := func(f *pflag.Flag) {
 			if namedOption, ok := f.Value.(opts.NamedOption); ok {
-				if _, valid := unknownKeys[namedOption.Name()]; valid {
-					delete(unknownKeys, namedOption.Name())
-				}
+				delete(unknownKeys, namedOption.Name())
```

Comment:
(sorry, there's no way to comment the commit message itself, so putting this here)

The commit message from this commit mentions issue that is not here (SA4003), please remove

Reply:
Good catch; copied a couple of lines too many. Updated 👍